### PR TITLE
Update requirements.md

### PR DESCRIPTION
**Pull request by offtherailz**
## Description
This PR adds to documentation requirements for database and system requirements, beause is not clearly indicated. 

I retrieved the information from here:

http://training.mapstore.geosolutionsgroup.com.s3-website-eu-west-1.amazonaws.com/administration/basics.html#requirements

Anyway I know recent changes to support higer versions of postgres.  @randomorder and @tdipisa can you please check if they are updated. @tdipisa you can also indicate if you want to backport this doc or not.

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: docs/developer-guide/requirements.md [hunk 1]
```diff
@@ -34,3 +34,21 @@ These tools needs to be installed (other than **Java** in versions above above):
...
     - ³ Python is only needed for building documentation.
+    
+ 
+## Running in Production 
+ 
+### System requirements
+
+| Resource  | Minimum | Recommended |
+|-----------|---------|-------------|
+| Processor | 1 Core  | 2 Core      |
```
Comment (randomorder) [inline docs/developer-guide/requirements.md:45]:
```suggestion
| Processor | 2 Core  | 2 Core      |
```

File: docs/developer-guide/requirements.md [hunk 2]
```diff
@@ -34,3 +34,21 @@ These tools needs to be installed (other than **Java** in versions above above):
...
+| Memory    | 1 GB    | 2 Gb        |
```
Comment (randomorder) [inline docs/developer-guide/requirements.md:46]:
```suggestion
| Memory    | 2 GB    | 4 GB        |
```

**Review by randomorder — CHANGES_REQUESTED** [review]
bumped the requirements a bit here are the Min reqs from our Docs
https://training.mapstore.geosolutionsgroup.com/administration/basics.html#requirements

File: docs/developer-guide/requirements.md [hunk 3]
```diff
@@ -34,3 +34,21 @@ These tools needs to be installed (other than **Java** in versions above above):
...
+| Processor | 2 Core  | 2 Core      |
+| Memory    | 2 GB    | 4 GB        |
+
+### Database
+
+In production a PostgreSQL database is recommended: 
+
+| Tool     | Link                                               | Minimum | Recommended | Maximum    |
+|----------|----------------------------------------------------|---------|-------------|------------|
+| Postgres | [link](https://www.postgresql.org/)                | 9.6     | 11          | 11         |
```
[the remark this answers is not in the document]
Comment (tdipisa) [reply docs/developer-guide/requirements.md:54]: The answer is yes, the reference issue is https://github.com/geosolutions-it/MapStore2/issues/7272. @offtherailz no need to backport this.

[the remark this answers is not in the document]
Comment (lpasquali) [reply docs/developer-guide/requirements.md:54]: I agree that also for the auth encryption defaults (more secure than "hackable" md5) we should recommend pg13

**Review by tdipisa — APPROVED** [review]

**Pull request merged**